Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -5568,6 +5569,29 @@ void WebPage::dragCancelled()
 
 #endif // ENABLE(DRAG_SUPPORT)
 
+#if ENABLE(MODEL_PROCESS)
+void WebPage::requestInteractiveModelElementAtPoint(WebCore::IntPoint clientPosition)
```

Comment:
Nit - you can omit the `WebCore::` here, since there's a `using namespace` above.